### DISCUSSION TITLE
tine podcast: 20 – Aufgaben

### DISCUSSION THREAD

**Discussion by pschuele** [tine-groupware/tine#49] [Announcements]
https://www.tine-groupware.de/podcast/20-aufgaben/

Herzlich willkommen zum tine Podcast! Wir möchten Euch zu einem Blick hinter die Kulissen der Open-Source-Groupware tine mitnehmen. Achtung! Manchmal kann es technisch werden.

In dieser Episode sind wir multi-thematisch unterwegs – aus Gründen. tine-Entwickler Philipp Schüle erklärt zunächst die Grundzüge der Aufgaben-Anwendung. Die ist vielseitig, aber doch recht klein, daher haben wir noch Zeit für einige Fragen und Themen, die zuletzt auftauchten: Was sind die Weeklies, wie werden die Releases benannt und was hat es mit der Sicherheitslücke auf sich, die kürzlich geschlossen wurde? Ganz nebenbei braucht Philipp Tipps von Euch und verrät weltexklusiv den Namen des nächsten Major Release.

Gebt uns gerne Feedback zu dieser Episode. Habt Ihr vielleicht Themen, über die wir unbedingt mal sprechen sollten oder andere Anregungen?  Wenn es Euch gefallen hat, hinterlasst uns doch eine positive Bewertung im Podcast-Portal Eurer Wahl.

Neue Episoden erscheinen immer am vierten Mittwoch des Monats – am besten richtet Ihr Euch schon mal einen Serientermin in tine ein.

**Comment by lab-at-nohl** [comment]
Highlight war wirklich das System der Release-Nummerierung :1st_place_medal:. Ich meine mich zu erinnern, dass bei den weekly releases die letzte Zahl die **Anzahl der Commits** benennt...

In Sachen Tasks-App für Android kenne ich auch nichts Herausragendes, insbesondere gar nichts, was ActiveSync könnte und nichts, was keinen zusätzlichen Kalender erzeugt. Brauche es aber auch nicht zu sehr.

**Reply by pschuele** [reply]
danke für die rückmeldung. an den weeklies sind wir noch dran - und es wird auch noch eine info-page in den docs zur versions-nummerierung geben ...

**Reply by pschuele** [reply]
die versionierung / releasematrix ist jetzt hier dokumentiert:
https://tine-docu.s3web.rz1.metaways.net/operators/Releases/
